pyproject.toml

```diff
@@ -1,8 +1,34 @@
 [tool.poetry]
 name = "sqlalchemy-ibmi"
-version = "0.1.0"
-description = "'SQLAlchemy support for IBM i db2'"
-authors = ["Naveen-Ram <naveen.ram@ibm.com>"]
+version = "0.9.0"
+description = "SQLAlchemy support for Db2 on IBM i"
+readme = "README.md"
+author = [
+    "Naveen Ram <naveen.ram@ibm.com>"
+    "Kevin Adler <kadler@us.ibm.com>"
+]
+license = "Apache-2.0"
+keywords = ["sqlalchemy", "database", "ibm", "ibmi", "db2"]
+classifiers =
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache License 2.0",
+    "Operating System :: OS Independent",
+    "Topic :: Databases :: Front-end, middle-tier"
+packages = [
+    {include = "sqlalchemy_ibmi"}
+]
+include = [
+    { path = "test/*", format = "sdist" },
+    { path = "contributing/*", format = "sdist" },
+    { path = "docs/*", format = "sdist" },
+    { path = "setup.cfg", format = "sdist" },
+    { path = "CHANGES", format = "sdist" },
+    { path = "LICENSE", format = "sdist" },
+]
```

Comment:
Just an FYI, this requires an unreleased version of Poetry. See https://github.com/IBM/python-itoolkit/commit/64a603df251ad51eb2dd94ba8d5e61debd28d576

Reply:
This also requires changing the build-system requires and build-backend to work correctly as poetry is being re-organized to work with PEP 517.

Reply:
Should this be left out for now then?

Reply:
Nah, we can leave it in. We just might not be able to use automation at the moment to release packages.

Reply:
We do need to adjust the build-system setting, though.